components/match2/wikis/criticalops/match_group_input_custom.lua

```diff
@@ -0,0 +1,602 @@
+---
+-- @Liquipedia
+-- wiki=criticalops
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local MathUtil = require('Module:MathUtil')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
+local Variables = require('Module:Variables')
+local Streams = require('Module:Links/Stream')
+
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local args = require('Module:Arguments')
```

Comment:
just no

Reply:
Should I add something else in place?

currently if local args removed, it throws **Lua error in Module:Table at line 545: attempt to index upvalue 'tbl' (a nil value).** Inside it mentions line 265:

![image](https://user-images.githubusercontent.com/88981446/218303858-c8192d0b-4780-445f-8945-ce7651a1cb43.png)

Reply:
use match instead of args

Reply:
Ok, fixed now. The last 2 thing would be now with scores coming back, it doesnt for a Bo3/5 series (but works for Bo1) and the unused **local winner** (or is it just **local winner** at the top?)

![image](https://user-images.githubusercontent.com/88981446/218304230-6700e870-83bd-426e-86c4-e0d090d49f17.png)

The revision of this is up in PR

Reply:
scores do not work because you removed to much
![Uploading 4EE2C098-7B99-4E9D-856F-14BA6222B4B3.jpeg…]()